platform/platform-app/src/main/resources/features.xml

```diff
@@ -837,6 +837,8 @@
         <feature>mime-core-api</feature>
         <bundle>mvn:ddf.mime.core/mime-core-impl/${project.version}</bundle>
         <bundle>mvn:ddf.mime.core/mime-core-configurableresolver/${project.version}</bundle>
+        <bundle>mvn:org.apache.tika/tika-core/${tika.version}</bundle>
+        <bundle>mvn:org.apache.tika/tika-parsers/${tika.version}</bundle>
```

Comment:
We are already including our own tika-bundle in the catalog features file, instead we can just include it here. I don't think we want multiple things exporting tika packages.